### PR TITLE
Genesis Challenge 1 Success page

### LINKED ISSUE #261: Add a success page for chapter 1, challenge 1, part 1
This celebrates the first successful completion of a task to encourage learners. It should show right after completing the first task (which is copying the ScriptSig HEX from block 0). See the [Figma design](https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=2374%3A26698&t=1WVTKCVc1kPVBDfc-1) and the [export-ready image](https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=2418%3A25721&t=1WVTKCVc1kPVBDfc-1).

image

### PR DESCRIPTION
**Pull request by vinayaksh42**
Fixes #261

### CODE REVIEW

File: i18n/locales/en/translations.ts [hunk 1]
```diff
@@ -290,4 +290,10 @@ export const translations = {
     paragraph_five:
       'Holocat: “You better get to work. I can help, but you have to start meow.”',
   },
+  genesis1success: {
```
Comment (afterburn) [inline i18n/locales/en/translations.ts:293]: genesis_one_success

File: content/lessons/index.tsx [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
 
 export const lessons = {
-  holocat: Holocat,
-  'genesis-1': Genesis1,
-  'genesis-2': Genesis2,
-  'genesis-3': Genesis3,
+  'genesis-1': Holocat,
+  'genesis-2': Genesis1,
+  'genesis-3': Genesis2,
+  'genesis-4': Genesis1Success,
```
Comment (afterburn) [inline content/lessons/index.tsx:15]: Can we rename these to `Genesis4` and `Genesis5`?

File: content/lessons/genesis/genesis-5/index.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import { CodeExample, LessonInfo, Title, Text, TerminalChallenge } from 'ui'
+
+export default function Genesis3({ lang }) {
+  const t = useTranslations(lang)
+
+  return (
+    <TerminalChallenge
+      expectedInput={{
+        userVariable: '$scriptSigHex',
+        value:
```
Comment (afterburn) [inline content/lessons/genesis/genesis-5/index.tsx:13]: Not sure if this is just github being weird, but formatting seems off here.

**Review by afterburn — CHANGES_REQUESTED** [review]
Thanks @vinayaksh42, requested a couple small changes.

File: content/lessons/genesis/genesis-5/index.tsx (hunk 3 above)
Comment (vinayaksh42) [reply content/lessons/genesis/genesis-5/index.tsx:13]: I have formatting on save and it seems to be running correctly, I think due to the length of the string it is formatting it to the next line

**Comment by ecurrencyhodler** [comment]
Looks like the back button in the top left still looks a bit off. I wonder if you forked the repo before we merged #277. 

If so might need to rebase.  Other than that this is a tACK.
image

**Review by afterburn — APPROVED** [review]

**Comment by afterburn** [comment]
> Looks like the back button in the top left still looks a bit off. I wonder if you forked the repo before we merged #277. 
> 
> If so might need to rebase.  Other than that this is a tACK.
> image

Merged this PR, lets address the back button issue in a separate one.

**Comment by ecurrencyhodler** [comment]
Checked prod. There is no more issue with back button logo. No issue needs to be created.

**Pull request merged**